src/mmw/js/src/compare/views.js

```diff
@@ -201,10 +205,21 @@ var CompareWindow2 = modalViews.ModalBaseView.extend({
             var selected =
                 activeTab.get('selections').findWhere({ active: true });
 
+
```

Comment:
Also committed by mistake, but this function's going to get removed when the water quality work is in.